frontend/src/components/molecules/DetailView.tsx

```diff
@@ -101,6 +125,19 @@ const DetailView: React.FC = () => {
             {developmentArea.name}
           </Button>
         ))}
+        {subGoals.length ? (
+          <Box>
+            {/* Todo: put ${selectedNode?.name} */}
+            <Heading as="h2" size="2xl" fontWeight="hairline">
+              DELMÅL:
+            </Heading>
+            <SimpleGrid columns={2}>
```

Comment:
Does this need to be a grid? Doesn't seem like SubGoalContainer will have a fixed size.

Reply:
Agreed, but to avoid large commits in branches we focused mainly on the functionality and would improve the design at a later stage.